README.md

```diff
@@ -1 +1,61 @@
-# Oop-library
```

Comment:
### Your Summary
![image](https://user-images.githubusercontent.com/93039359/195207656-d8707b53-084d-4273-9d4c-6d5d43ee7e94.png)

### Descriptive Summary Example :heavy_check_mark:
![image](https://user-images.githubusercontent.com/93039359/195207786-c87dc56e-7619-42cb-8dd9-cafbd2d01bd6.png)

- [ ] You have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. It would be nice if you add more details in it as you can see in the second image above. :thinking: